src/pages/Login.jsx

```diff
@@ -8,7 +8,7 @@ const Login = () => {
   return (
     <Container>
         <Header>
-            <img src={logo}/>
+            <img src={logo} alt=""/>
```

Comment:
Please change alt text to "Site logo" here, it will be useful for people with disabilities.